cpp/include/cugraph/edge_property.hpp

```diff
@@ -126,6 +127,26 @@ class edge_dummy_property_t {
   auto view() const { return edge_dummy_property_view_t{}; }
 };
 
+template <typename GraphViewType, typename T>
+auto create_constant_edge_property(raft::handle_t const& handle,
+                                   GraphViewType const& graph_view,
+                                   T constant_value)
+{
+  edge_property_t<GraphViewType, T> edge_property(handle, graph_view);
+
+  auto mutable_view = edge_property.mutable_view();
+
+  std::for_each(
+    thrust::make_zip_iterator(mutable_view.value_firsts().begin(),
+                              mutable_view.edge_counts().begin()),
+    thrust::make_zip_iterator(mutable_view.value_firsts().end(), mutable_view.edge_counts().end()),
+    [&handle, constant_value](auto tuple) {
+      detail::scalar_fill(handle, thrust::get<0>(tuple), thrust::get<1>(tuple), constant_value);
+    });
+
+  return edge_property;
+}
```

Comment:
We have `fill_edge_src|dst_property()` (https://github.com/rapidsai/cugraph/blob/branch-23.06/cpp/src/prims/fill_edge_src_dst_property.cuh).

So, it will be more consistent to create `fill_edge_property()` and call this instead of adding this function. I can quickly write `fill_edge_property()` if you agree.

Reply:
Merged your PR and then my change to use it.

Reply:
Hang on... pushed to wrong branch.